dev-docs/analytics/intentiq.md

```diff
@@ -26,3 +26,61 @@ pbjs.enableAnalytics({
     provider: 'iiqAnalytics'
 });
 ```
+
+### Manual Report Trigger with reportExternalWin
+
+The reportExternalWin function allows for manual reporting, meaning that reports will not be sent automatically but only when triggered manually.
+
+To enable this manual reporting functionality, you must set the manualWinReportEnabled parameter in Intent IQ Unified ID module configuration is true. Once enabled, reports can be manually triggered using the reportExternalWin function.
```

Comment:
Where is this manual thing set? Looks like it may be in the usersync config? It would clearer with an example here.

Reply:
Hi @bretg, the document has been updated to include a code example.